python/src/aiconfig/registry.py

```diff
@@ -61,7 +61,7 @@ def get_model_parser(model_id: str) -> ModelParser:
         Returns:
             ModelParser: The retrieved model parser
         """
-        return ModelParserRegistry._parsers[model_id]
+        return ModelParserRegistry._parsers[model_id.strip()]
```

Comment:
For example, here is a "get" instead of set, so we probably don't need to change this?

Reply:
The problem is that this function `get_model_parser()` is an external public call, so it's possible for someone to call into it with unsanitized trailing and leading whitespace

Reply:
Doesn't that contradict the PR summary, though:
> Because we could have existing configs that have model names already, I only changed the setter methods, not the getter ones like get_model(), so this is not a breaking change

Reply:
Yea you're right, let me re-word the the summary.